Index: Pizzeria/routes/web.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?php\r\n\r\nuse Illuminate\\Support\\Facades\\Route;\r\n\r\n/*\r\n|--------------------------------------------------------------------------\r\n| Web Routes\r\n|--------------------------------------------------------------------------\r\n|\r\n| Here is where you can register web routes for your application. These\r\n| routes are loaded by the RouteServiceProvider within a group which\r\n| contains the \"web\" middleware group. Now create something great!\r\n|\r\n\r\nRoute::get('/', function () {\r\n    return view('welcome');\r\n});\r\n*/\r\nRoute::resource('products',ProductsController::class);\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Pizzeria/routes/web.php b/Pizzeria/routes/web.php
--- a/Pizzeria/routes/web.php	(revision 593cfce2a32989c727d9cb38dc4230746da57fc7)
+++ b/Pizzeria/routes/web.php	(date 1632821066869)
@@ -13,7 +13,11 @@
 |
 
 Route::get('/', function () {
-    return view('welcome');
 });
 */
+Route::get('/pizzas/dodaj-skladnik/{id}', 'PizzasController@dodajSkladnik');
+Route::get('/pizzas/usun-skladnik/{id}', 'PizzasController@usunSkladnik');
 Route::resource('products',ProductsController::class);
+Route::resource('pizzas',PizzasController::class);
+
+
